### PR TITLE
 blockdev_mirror_with_ignore:mirror with target error ignore

### PR DESCRIPTION
**Pull request by aliang123**
Mirror with on-target-error "ignore"

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2002566

### CODE REVIEW

**Comment by aliang123** [comment]
PASS 1-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_mirror_with_ignore.q35

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

File: qemu/tests/blockdev_mirror_with_ignore.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    def blockdev_mirror(self):
+        super(BlockdevMirrorWithIgnore, self).blockdev_mirror()
+        timeout = self.params.get("job_timeout", 600)
+        for job_id in self._jobs:
+            get_event = job_utils.get_event_by_condition
+            event = get_event(self.main_vm, job_utils.BLOCK_JOB_ERROR_EVENT,
+                              timeout, device=job_id, action='ignore')
+            if not event:
+                self.test.fail("Mirror job can't reach error after %s seconds",
```
Comment (zhencliu) [inline qemu/tests/blockdev_mirror_with_ignore.py:18]: test.fail takes one arg, so it should be self.test.fail('xxxx %s' % timeout)

Comment (aliang123) [reply qemu/tests/blockdev_mirror_with_ignore.py]: Have updated, thanks zhenchao.

**Comment by aliang123** [comment]
PASS 1-Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_commit_with_ignore.q35

**Comment by ngu-niny** [comment]
PASS 1-Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_mirror_with_ignore

Acked-by: Nini Gu <ngu@redhat.com>

**Comment by aliang123** [comment]
PASS 1-Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_mirror_with_ignore.q35

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**